### DISCUSSION TITLE
My dumb mistake leading to many issues..

### DISCUSSION THREAD

**Discussion by temporary-simon** [community/community#58140] [Other Feature Feedback, Questions, & Ideas]
### Dumb mistake (a virus) leading to suspensions across all platforms

General

### Body

Hey, there GitHub Community,

I hope this finds you well. I'm reaching out because I've had a tough time recently with my GitHub account and need your help to sort things out.

So, here's what happened. A couple of weeks back, I downloaded a program from GitHub. My Windows Defender popped up a "potential virus" warning, but I dismissed it thinking it was because the software wasn't an "official" program. In hindsight, not the smartest move, I admit.

Fast forward a week, and my Facebook gets deactivated. Facebook claimed it was due to an issue with my Instagram. After checking Instagram, I saw a new account had been linked to mine - an account I'd never created. I figured something was up, so I deleted that account from my "account center" immediately and reported it to Facebook. I shot off a bunch of emails to their support, hoping they could help me figure out what was going on.

But then, for a whole week, I heard nothing - nada. I was checking my email every day, hoping for some kind of update, but nothing came leaving me pretty hopeless about receiving a response. Also after reading articles online about Facebook and their great corporation don't answer small fish like me, but suddenly a week after just like that, my Facebook account was back. No warning, no explanation. I was just... back in. Usually, Facebook is pretty good at letting me know when something weird's going on with my account, so this was really confusing. (Like "X has logged in from Y ...")

Was it because I deleted the suspicious Instagram account? Did my emails to Facebook finally get through? I honestly have no idea. But hey, at least my account was back.

Just when I thought things were settling down, my Reddit account was banned out of the blue a couple of days later. At this point, it was clear that something fishy was going on, so I went ahead and changed all my important passwords - well, all except GitHub, where I of course didn't have 2FA set up either. Yep, you guessed it - oversight number two.

A few days after that, my GitHub account got suspended. This I realized when my friends and I hit a snag with a repo we're using for an upcoming exam. Plus, I found this odd repo on my GitHub, a carbon copy of that pesky virus file I had downloaded. I'm guessing that's what triggered the suspension. But, believe me, I had no intention of spreading a virus.

Just to be clear, I've been using GitHub since 2016 and have always followed the rules. I would never knowingly engage in any activities that could potentially harm others or violate GitHub's policies. To top it all off, my "stars" tab started showing way more stars than it used to - like 30-40 instead of the usual 11-12. It seemed like a bunch of other folks had been hit by the same problem.

And also to just summarize all my accounts that have been banned over the course of like 2 weeks (Facebook, Instagram, Reddit, and GitHub.) and I've gotten most of them back but GitHub is the most important one for me.

This whole thing's been really stressful, especially with an important exam coming up on June 22. I just want to get my account unsuspended and get back to normal. I tried emailing you guys already but didn't hear back. I'm really hoping you can help me out here.

Thanks for taking the time to read my story. I'm looking forward to sorting this all out soon.

And last but not least, I can obviously prove all this I have emails, pictures, etc I can show the notifications on Instagram popping up as well as pictures of me being banned on different platforms and then being unbanned again. I just don't like posting them "online" in a forum so I'd rather send or screenshare in a more private manner so I don't leak more than I have already done. 

Best,

Simon

**Comment by airtower-luna** [comment]
I'm afraid that's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. From what I heard tickets can take a while though, so it might be best to look for other ways to prepare your exam. :confused: 

Either way, good luck!

**Reply by temporary-simon** [reply]
Yes, that's what I was a bit worried about. I just read on the GitHub documentation, somewhere that the GitHub community consisted of both staff and community members, so I decided to just give it a shot! 

The worst thing about it is my work also uses GitHub so I had to remove the suspended user and invite this temporarily so I could do my job, and even then my GitKraken only had a 7-day trial which is over-due, so I had to relearn Git CLI for everything, which takes a time to get used to if you're used to using a GUI 😩🙈

 Anyway, thanks a lot for commenting, I'll try and find another way, thanks. 😩😩
